.github/workflows/release.yaml

```
@@ -23,3 +24,6 @@ jobs:
         Expire-Date: 0
         EOF
         DRY_RUN=true ./scripts/release.sh --no-upload --no-docker-push --in-place 3.6.99
+    - name: test-image
+      run: |
+        make test-images
```

It seems the target test-image isn't necessary. 
```suggestion
        make test-images
        VERSION=3.6.99 ./scripts/test_image.sh
```